olp-cpp-sdk-core/include/olp/core/http/NetworkUtils.h

```diff
@@ -38,6 +40,15 @@ class NetworkUtils {
   static size_t CaseInsensitiveFind(const std::string& str1,
                                     const std::string& str2, size_t offset = 0);
 
+  /**
+   * @brief Extract the user agent from the headers. User agent is removed from
```

Comment:
Proper Doxygen. I am pretty sure @HalynaDumych will be mad at you for not doing this right!